docs/containers-authfile.5.md

```diff
@@ -0,0 +1,57 @@
+% containers-authfile(5)
+
+# NAME
+containers-authfile - syntax for the registry authentication file
+
+# DESCRIPTION
+
+A credentials file stored at `${XDG_RUNTIME_DIR}/containers/auth.json` in
+json format used to authenticate against container image registries.
+
+## FORMAT
+
+The auth.json file stores encrypted authentication information for the
+user to container image registries.  The file can have zero to many entries and
+is created by a `login` command from a container tool such as `podman login` or
+`buildah login`.  Each entry includes the name of the registry and then an auth
+token in the form of a base64 encoded string from the concatenation of the
+username, a colon, and the password.
+
+The following example shows the values found in auth.json after the user logged in to
+their accounts on quay.io and docker.io:
+
+```
+{
+	"auths": {
+		"docker.io": {
+			"auth": "erfi7sYi89234xJUqaqxgmzcnQ2rRFWM5aJX0EC="
+		},
+		"quay.io": {
+			"auth": "juQAqGmz5eR1ipzx8Evn6KGdw8fEa1w5MWczmgY="
+		}
+	}
+}
+```
+
+An entry can be removed by using a `logout` command from a container
+tool such as `podman logout` or `buildah logout`.
+
+In addition, a Docker credential store can be created and the Docker credentials-helper software can be used to manage the credentials in a more secure way than depending on the base64 encoded authentication provided by `login`.
+
+When the Docker credentials-helper is in use on a Linux platform, the auth.json file would contain:
+
+```
+    "auths": {
+        "localhost:5001": {}
+    },
+    "credsStore": "secretservice"
```

Comment:
@QiWang19 @mtrmac Do we support credsStore?

Reply:
No. I see the code can only parse "credHelpers":{}

Reply:
So we should not document it here, until we get containers/image to support credStore.